src/ucp/proto/proto_am.inl

```diff
@@ -86,6 +122,13 @@ ucs_status_t ucp_do_am_bcopy_multi(uct_pending_req_t *self, uint8_t am_id_first,
             packed_len = uct_ep_am_bcopy(uct_ep, am_id_first, pack_first, req, 0);
             UCS_PROFILE_REQUEST_EVENT_CHECK_STATUS(req, "am_bcopy_first",
                                                    packed_len, packed_len);
+            if (ucs_unlikely(packed_len == UCS_ERR_NO_RESOURCE) &&
```

Comment:
if `ucp_am_handle_pending_req_header` handles status, why need to check `UCS_ERR_NO_RESOURCE` before?